doc/docusaurus/docs/essential-concepts/versions.md

```diff
@@ -0,0 +1,68 @@
+
+It is only when you write a Plutus _validator script_, put it in a transaction and submit it to the Cardano ledger, that ledger language version becomes relevant. It is relevant in three ways:
+
+- Depending on the ledger language version, the ledger will pass different arguments to the validator.
+  For instance, the arguments passed to a Plutus V2 validator include a list of reference inputs of the transaction. This is not passed to a Plutus V1 validator since reference inputs were introduced later than Plutus V1 (specifically, Plutus V1 was introduced in the Alonzo era, while reference inputs were introduced in the Babbage era).
+  Similarly, only Plutus V3 can be used to validate governance actions, since governance actions are introduced in the Conway era and are not part of the pre-Conway era transactions, and thus aren't part of the arguments passed to Plutus V1 or V2 validators.
+- At present, a newer ledger language version has access to more builtin functions than an older ledger language version.
+  Bear in mind that this will change as we plan to make all builtin functions available to all ledger language versions.
+- At present, Plutus Core version (explained below) 1.1.0 can only be used with Plutus V3, while Plutus V1 and V2 validators must use Plutus Core 1.0.0.
+  This will also change, as we plan to make all Plutus Core versions compatible with all ledger language versions.
+
+When do we introduce a new ledger language version?
+A new ledger language version must be introduced when a new ledger era is rolled out, because in a new ledger era, transactions will be different and contain new and/or modified fields, such as governance actions introduced in the Conway era.
+As a result, new information will need to be passed as arguments to Plutus validators.
+This necessitates a new ledger language version, because the arguments passed to existing ledger language versions must stay exactly the same, so that existing validators continue to work.
+
+Previously it was also the case that introducing new builtin functions or new Plutus Core versions necessitate a new ledger language version, but this is no longer the case.
+Introducing a new ledger language version is very expensive as it must be maintained forever.
+Going forward we expect the launch of new ledger eras to be the only reason for introducing new ledger language versions.
+
+## Plutus Core Version
+
+Plutus Core version is the usual sense of version pertaining to programming languages - in this instance the Plutus Core language.
+So far there have been two Plutus Core versions: 1.0.0 and 1.1.0.
+1.1.0 adds sums-of-products to the language by introducing two new AST node types: `Case` and `Constr`.
+See CIP-85 for more details.
+
+Note that adding new builtin functions does not require a new Plutus Core version.
+Once a new builtin function is added, one can simply start using the new builtin function with an existing Plutus Core version.
```

Comment:
So extending the semantics of UPLC does not warrant a new language version. Would it be enough to say that any changes to UPLC which are not backwards compatible create a new language version? Then again, why did we need a new language version for SoPs? Doesn't that only extend the language? Or does it also make some backwards-incompatible changes?

Reply:
> Would it be enough to say that any changes to UPLC which are not backwards compatible create a new language version?

What do you mean by "language version"? I'm using the terms "ledger language version" and "Plutus Core version" here. Any changes to UPLC itself, whether backwards compatible or not, creates a new Plutus Core version, but does not create a new ledger language version.

Reply:
So what is the exact relationship between "ledger language" versions and "Plutus Core" versions?

Reply:
A new Plutus Core version changes the languages - e.g., Plutus Core 1.1.0 introduces `Case` and `Constr`.

A new ledger language version does not change the language. The main difference is that the scripts will receive different arguments from the ledger.

Reply:
I understand. I think it would be very helpful to add this to the document as an example. Thank you!

Reply:
The paragraph above does say "1.1.0 adds sums-of-products to the language by introducing two new AST node types: `Case` and `Constr`.".